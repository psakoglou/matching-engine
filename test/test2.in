BUY IOC 100 10 ORDER1
SELL GFD 1000 130 ORDER2
BUY IOC 1001 120 ORDER3
BUY GFD 1002 10 ORDER4