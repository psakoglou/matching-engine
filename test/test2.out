TRADE ORDER3 1001 120 ORDER2 1000 120
TRADE ORDER4 1002 10 ORDER2 1000 10